unhuf.py

```diff
@@ -22,6 +25,14 @@ def get_args() -> argparse.Namespace:
 
     parser.add_argument("files", metavar="FILE", nargs="+", type=Path)
 
+    parser.add_argument(
+        "-t",
+        "--type",
+        choices=TYPE_CHOICES,
+        default=TYPE_CHOICES[0],
+        help="Choose which type of the algorithm will be used",
+    )
+
```

Comment:
Why is this type selection added to decoding? Info about which algorithm was used to encode file should be read from it. These algorithms are not cross compatible for decoding, right?